lib/pengine/complex.c

```diff
@@ -228,26 +228,37 @@ get_meta_attributes(GHashTable * meta_hash, pcmk_resource_t * rsc,
     }
 }
 
+/*!
+ * \brief Get final values of a resource's instance attributes
+ *
+ * \param[in,out] meta_hash  Where to store values
```

Comment:
Probably better not to have "meta" in the name

Reply:
good point